src/main/python/covata/delta/apiclient.py

```diff
@@ -375,6 +394,76 @@ def get_events(self, requestor_id, secret_id=None, rsa_key_owner_id=None):
         response.raise_for_status()
         return response.json()
 
+    @utils.check_id("requestor_id")
+    @utils.check_optional_id("base_secret_id, created_by, rsa_key_owner_id")
+    @utils.check_arguments(
```

Comment:
page and page size are used in a few places - so perhaps we can turn this into a util method.